src/code42cli/cmds/cases.py

```diff
@@ -204,9 +213,9 @@ def file_events_list(state, case_number, format):
     formatter = OutputFormatter(format, _get_events_header())
     try:
         response = state.sdk.cases.file_events.get_all(case_number)
-    except Py42NotFoundError as err:
-        click.echo("Invalid case-number.")
-        raise err
+    except Py42NotFoundError:
+        raise Code42CLIError("Invalid case-number.")
+
     if not response["events"]:
         click.echo("No events found.")
     for event in response["events"]:
```

Comment:
You are looping through the events twice here.

You can get rid of the `for` loop
Else we are printing a lot of extra

Reply:
Or perhaps this is supposed to loop through pages, if that is the case, then the for loop is incorrect, the first `event` never gets used..

Do you mean to do this:

```
for page in response:
          events = [event for event in page["events"]]
          formatter.echo_formatted_list(events)
```

Reply:
I recommend IDEs like PyCharm, it points out mistakes like this whereas text editors do not typically